lib/resty/core/response.lua

```diff
@@ -45,6 +45,57 @@ ffi.cdef[[
 ]]
 
 
+local ngx_lua_ffi_set_resp_header
+
+ngx_lua_ffi_set_resp_header = function(r, key, key_len, is_nil,
```

Comment:
This can be put inside the `else` branch of `if MACOS_ARM64 then`, otherwise we seems to be generating unused GCfunc in macOS for no apparent benefit.